openapi.yaml

```diff
@@ -1124,6 +1170,129 @@ paths:
               schema:
                 $ref: '#/components/schemas/ForbiddenError'
           description: ''
+  /accounts/v2/user_satisfaction_survey/show_survey_prompt/:
+    get:
+      operationId: show_survey_prompt
+      summary: Show survey prompt
+      parameters:
+      - in: header
+        name: X-Workspace
+        schema:
+          type: string
+        description: Organization unique_key
+        required: true
+      tags:
+      - user_satisfaction_survey
+      security:
+      - SessionAuthentication: []
+      responses:
+        '200':
+          content:
+            application/json:
+              schema:
+                type: object
+                additionalProperties: {}
+                description: Unspecified response body
+          description: ''
+        '401':
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/UnauthenticatedError'
+          description: ''
+        '403':
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/ForbiddenError'
+          description: ''
+  /accounts/v2/user_satisfaction_survey/update_g2_review_updated_on/:
+    post:
+      operationId: update_g2_review_updated_on
+      summary: Update g2_review_updated_on
+      tags:
+      - user_satisfaction_survey
+      security:
+      - SessionAuthentication: []
+      responses:
+        '200':
```

Comment:
should be 204

Reply:
Updated in main of appserver.

@deekshith1856 , delete this pr and create new one.